build/arch/PKGBUILD-git

```diff
@@ -44,5 +41,5 @@ build() {
 package() {
   cd "$pkgname"
 
-  make DESTDIR="$pkgdir/" install
+  make PREFIX="/usr" DESTDIR="$pkgdir/" install
```

Comment:
I wonder why you put PREFIX to Makefile? Have you tested it? It doesn't work on my Arch machine